src/roles/common/templates/nuhs.j2

```diff
@@ -71,10 +71,25 @@ cpuset:
 image_path: {{ image_path | default("MISSING") }}
 image_file_name: {{ image_file_name | default("MISSING") }}
 nuh_timezone: {{ item.timezone | default("America/Toronto") }}
-install_ntp: {{ item.install_ntp | default(true) }}
 skip_vsd_installed_check: {{ item.skip_vsd_installed_check | default(false) }}
 health_monitoring_agent: {{ item.health_monitoring_agent | default("none") }}
 
+vsdconfig_enable: {{ nuh_common.vsdconfig_enable | default(false) | lower }}
+elasticsearch_enable: {{ nuh_common.elasticsearch_enable | default(false) | lower }}
+
+{% if nuh_vrrp is defined and nuh_vrrp %}
+vrrp:
+{% for vr in nuh_vrrp %}
+  - router_id: {{ vr.router_id }}
+    interface: {{ vr.interface }}
+    vip:
+{% for vip_address in vr.vip %}
```

Comment:
What happens if there is an empty list for vip?

Reply:
Added check for empty